econml/_cate_estimator.py

```diff
@@ -860,7 +861,7 @@ def _expand_treatments(self, X=None, *Ts, transform=True):
                 T = np.full((n_rows,) + self._d_t_in, T)
 
             if self.transformer and transform:
-                if not (hasattr(self, 'treatment_featurizer') and self._original_treatment_featurizer):
+                if not self._original_treatment_featurizer:
                     T = T.reshape(-1, 1)
```

Comment:
Yes, that makes perfect sense, thanks.  But please make sure that at least one test covers the multiple-treatments with a featurizer use case in case that's not already being done.

Reply:
I'll create a test for multiple treatments passed to a treatment featurizer